go.mod

```diff
@@ -3,13 +3,12 @@ module github.com/shermp/Kobo-UNCaGED
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/bamiaux/rez v0.0.0-20170731184118-29f4463c688b
+	github.com/geek1011/koboutils/v2 v2.0.0
 	github.com/gofrs/uuid v3.2.0+incompatible
```

Comment:
You might want to update this to `master`, as I've loosened up the `.kobo/version` parsing (this mainly will be useful for manually-edited version files; nickel will never produce badly-formed version files itself).

Reply:
My question is, who on earth modifies their version file?

Reply:
Me! I did it when testing some device-specific stuff.